src/app/index.tsx

```diff
@@ -107,15 +108,15 @@ export default (options: AppOptions) => {
     reducer,
   });
 
-  const container = () => (
+  const container: React.FC<{}> = () => (
```

Comment:
it shouldn't really need the type at all on the declaration, but in any case i'm pretty sure the empty props is the default
```suggestion
  const container: React.FC = () => (
```